department-of-veterans-affairs/vets-website
# Bah s rzipcode 1655

## Description
Fixed SR for zip code message on windows jaws and NVDA

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/1647

## Testing done
- Testing done with Apple voice over on Mac and with NVDA and JAWS in Windows. 
- Go to http://localhost:3001/gi-bill-comparison-tool/profile/11929140 on Mac
- Go to http://10.0.2.2:3001/gi-bill-comparison-tool/profile/11929140 on Windows
- Go to Where will you take the majority of your classes? select other and expect the message "Your zip code is located in "what ever city you enter"".
## Screenshots


## Acceptance criteria
- [x] SR only reads Your 'zip code is located in "what ever city you enter"' once. 

## Definition of done
